test/unit.test.js

```diff
@@ -1,9 +1,22 @@
 const fs = require('fs');
-const { join } = require('path');
+const { join, isAbsolute, resolve } = require('path');
 const { nodeFileTrace } = require('../out/node-file-trace');
 
 global._unit = true;
 
+const nodeBuiltins = [...require('repl')._builtinLibs, 'constants', 'module', 'timers', 'console', '_stream_writable', '_stream_readable', '_stream_duplex', 'process', 'sys'];
+const resolveDependency = (id, parent, ...rest) => {
```

Comment:
This seems too complex for the test. Maybe just do something like:

```js
const resolveDependency = (id, parent) => `custom-resolution-${id}-${parent}`;
```

Reply:
fair 👍